css/about.css

```diff
@@ -0,0 +1,62 @@
+html {
+    max-width: 960px;
+    /* justify-content; */
+    margin: auto;
+    /* margin-left: 30px; */
+}
+
+.head {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: baseline;
+}
+
+.upper a {
+    margin: 50px;
+    color: black;
+    text-decoration: none;
+}
+
+.topContent {
+    border-bottom: 3px black dotted;
+    margin-bottom: 15px;
+    padding: 25px;
+    margin: 10px;
+}
+
+.middle1, .middle2 {
+    display: flex;
+    flex-wrap: wrap;
+    border-bottom: 2px black dotted;
+    padding: 25px;
+    margin: 20px;
+}
```

Comment:
you can add in the justify-content: space-between or space-around or space-evenly here to push the cards/containers away from each other and give the white space in between them